recipes/triqs_ctseg/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "3.3.0" %}
+
+package:
+  name: triqs_ctseg
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/ctseg/releases/download/{{ version }}/ctseg-{{ version }}.tar.gz
+  sha256: 2fc8c358e339b22d40c7d8b8b60f2a6f61bce786045f3fe0831b86447e2e9c8f
+
+build:
+  number: 0
+  skip: true  # [win or py<30]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - wget                                   # [target_platform == 'osx-arm64']
+    - libtool                                # [target_platform == 'osx-arm64']
+    - autoconf                               # [target_platform == 'osx-arm64']
+    - automake                               # [target_platform == 'osx-arm64']
+    - llvm-openmp                            # [target_platform == 'osx-arm64']
```

Comment:
Why are these specific to `osx-arm64` and not for all osx-* variants?

Reply:
Thank you @xhochy for pointing this out, these lines remained from our osx-arm64 porting experiments and should be redundant now. Pushed the removal and awaiting CI results.